Meraki scenario guide

kbreit opened the pull request:
##### SUMMARY
A scenario guide for the Meraki modules will provide the following benefits:

- Explain what can be done with the modules
- Provide inspiration for what can be done programmatically with Meraki
- Explain limitations of the API and modules

##### ISSUE TYPE
 - Docs Pull Request

##### COMPONENT NAME
meraki

kbreit committed:
Initial commit for the Meraki scenario guide

dmsimard commented on docs/docsite/rst/scenario_guides/guide_meraki.rst:
```diff
@@ -0,0 +1,75 @@
+
+Meraki modules provide a user-friendly interface to manage your Meraki environment using Ansible. For example, details about SNMP settings for a particular organization can be discovered using the module `meraki_snmp <meraki_snmp_module>`.
+
+.. code_block:: yaml
```
Try this maybe ?
```
::
    - name: Query SNMP settings
      meraki_snmp:
          api_key: abc123
          org_name: AcmeCorp
          state: query
          delegate_to: localhost
```
I couldn't find any instance of ``code_block`` by grepping the docs.

kbreit replied:
Unscores and dashes are important.

kbreit committed:
Added Meraki guide to indexes, fixed an error

kbreit committed:
Added common parameters to scenario guide

kbreit committed:
Add additional information for first draft
- Added very common parameters everyone uses.
- Documented common format for returned data from Meraki.
- High level explanation of error handling.

kbreit committed:
Fix .rst formatting error

kbreit committed:
Added section about handling returned data. More to come.

dagwieers commented:
I don't know how I missed this. Should we merge this and backport this to v2.7 ?

kbreit commented:
We should. I am going to review it and make sure it's good before we merge and backport though. I should be able to do it in a day or two.

kbreit committed:
Small formatting changes

kbreit commented:
@dagwieers Okay lets merge this and I’ll get a back port into 2.7.

dagwieers commented on docs/docsite/rst/scenario_guides/guide_meraki.rst:
```diff
@@ -0,0 +1,128 @@
+
+Meraki and its related Ansible modules return most information in the form of a list. For example, this is returned information by ``meraki_admin`` querying administrators. It returns a list even though there's only one.
+
+.. code-block:: yaml
```
I would make this 'json' instead.

kbreit replied:
I'm trying to do this in the backport and really struggling. Is json not a valid option or do I need to include quotes?
